lib/connector.js

```diff
@@ -38,15 +42,17 @@ Connector.prototype.getDefaultIdType = function() {
  * @returns {Object} The metadata object
  * @property {String} type The type for the backend
  * @property {Function} defaultIdType The default id type
- * @property {Boolean} [isRelational] If the connector represents a relational database
+ * @property {Boolean} [isRelational] If the connector represents a relational
+ * database
  * @property {Object} schemaForSettings The schema for settings object
  */
-Connector.prototype.getMedadata = function () {
+Connector.prototype.getMetadata = function() {
   if (!this._metadata) {
     this._metadata = {
       types: this.getTypes(),
       defaultIdType: this.getDefaultIdType(),
-      isRelational: this.isRelational || (this.getTypes().indexOf('rdbms') !== -1),
+      isRelational: this.isRelational ||
+      (this.getTypes().indexOf('rdbms') !== -1),
```

Comment:
I would personally indent the next line of a multi-line statement by two spaces to make it more clear that L55 is a continuation of L54, not a next object property.

```js
isRelational: this.isRelational ||
  (this.getTypes().indexOf('rdbms') !== -1),
schemaForSettings: {
```

Even better, you can extract a helper method implementing `indexOf` lookup and thus fit the whole statement on a single line:

```js
isRelational: this.isRelational || this.hasType('rdbms'),
```